migration/migrator/data/submitty_db.sql

```diff
@@ -646,6 +646,27 @@ CREATE TABLE public.terms (
 );
 
 
+--
+-- Name: unverified_users; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.unverified_users (
+    user_id character varying NOT NULL,
+    user_numeric_id character varying,
+    user_givenname character varying NOT NULL,
+    user_preferred_givenname character varying,
+    user_password character varying,
+    user_familyname character varying NOT NULL,
+    user_email_secondary character varying(255) DEFAULT ''::character varying NOT NULL,
+    user_email_secondary_notify boolean DEFAULT false,
+    user_pronouns character varying(255) DEFAULT ''::character varying,
+    user_preferred_familyname character varying,
+    user_email character varying NOT NULL,
+    verification_code character varying(50) DEFAULT 'none'::character varying NOT NULL,
+    verification_expiration timestamp without time zone DEFAULT CURRENT_TIMESTAMP
+);
```

Comment:
Instead of collecting all of this information up front, would it make sense to only collect an email and perform email verification first?

Reply:
I have removed all of the unnecessary values from this, (preferred name, secondary email, pronouns, etc)